### ISSUE TITLE
Errors in section Neural Networks in Keras

### ISSUE THREAD

**Issue by miquelbeltran** [udacity/dlnd-issue-reports#587]
In https://classroom.udacity.com/nanodegrees/nd101/parts/73a7fe8e-4354-4362-939d-a8bf2bae870d/modules/e6a4e4a1-98a9-4afe-a744-7ae6a59c01b8/lessons/7c042ed0-08e7-4138-ad78-1a8d671d5da5/concepts/7c3d9b4c-206d-4430-9459-c78357181d9a (Neural Networks in Keras)

In the example provided it should be using `binary_crossentropy` instead of `categorical_crossentropy` and `evaluate` should be called with the parameters `X, y` like `model.evaluate(X, y)`

The quiz at the bottom is also wrong, what the quiz asks is not the same as the provided solution, check the description vs. the network_solution.py. 

The only way I could pass the quiz was by increasing the default 50 epochs to 500, then my model accuracy was 100% (minimum of 75% is required).
